src/services/gateway.service.ts

```diff
@@ -238,9 +238,9 @@ function updateOrganization({ name, country, size }: OrganizationOptions) {
   })
 }
 
-function fetchAllUsers(appId: AppId) {
+function fetchAllUsers(appId: AppId, offset: number, count: number) {
   return gatewayAuthorizedInstance.get(
```

Comment:
Is there a limit on number of users returned by one API call? We don't have any checks here on `count`.

Reply:
Currently there is no limit, so we are fetching all the users at once. In future we'll add scroll based pagination